include/arch/zynq7000.h

```diff
@@ -52,7 +52,15 @@ enum {
 
 typedef struct {
 	enum { pctl_set = 0, pctl_get } action;
-	enum { pctl_ambaclock = 0, pctl_devclock, pctl_mioclock, pctl_mio, pctl_devreset, pctl_reboot } type;
+	enum {
+		pctl_ambaclock = 0,
+		pctl_devclock,
+		pctl_mioclock,
+		pctl_mio,
+		pctl_devreset,
+		pctl_reboot,
+		pctl_sdwpcd
+	} type;
```

Comment:
We don't agree with clang-format on this matter (in the kernel code especially)
```suggestion
	enum { pctl_ambaclock = 0, pctl_devclock, pctl_mioclock,
		pctl_mio, pctl_devreset, pctl_reboot, pctl_sdwpcd } type;
```

Reply:
I added a `/* clang-format off */` comment to turn off formatting for these enums.